app/src/main/res/values/strings.xml

```diff
@@ -1,58 +1,58 @@
 <resources xmlns:xliff="urn:oasis:names:tc:xliff:document:1.2">
 
-    <string name="app_name">Seedvault</string>
+    <string name="app_name" translatable="false">Seedvault</string>
     <string name="backup">Backup</string>
 
     <string name="restore_backup_button">Restore backup</string>
 
     <!-- Settings -->
     <string name="settings_backup">Backup my data</string>
-    <string name="settings_backup_location">Backup location</string>
-    <string name="settings_backup_location_picker">Choose backup location</string>
-    <string name="settings_backup_location_title">Backup Location</string>
-    <string name="settings_backup_location_invalid">The chosen location can not be used.</string>
+    <string name="settings_backup_location">Backup folder</string>
+    <string name="settings_backup_location_picker">Choose where to save the backup</string>
+    <string name="settings_backup_location_title">Backup folder</string>
+    <string name="settings_backup_location_invalid">The chosen folder can not be used.</string>
     <string name="settings_backup_location_none">None</string>
-    <string name="settings_backup_location_internal">Internal Storage</string>
+    <string name="settings_backup_location_internal">Internal storage</string>
     <string name="settings_backup_last_backup_never">Never</string>
-    <string name="settings_info">All backups are encrypted on your phone. To restore from backup you will need your 12-word recovery code.</string>
+    <string name="settings_info">All backups are encrypted by your device. To restore one you need your 12-word recovery code.</string>
     <string name="settings_auto_restore_title">Automatic restore</string>
-    <string name="settings_auto_restore_summary">When reinstalling an app, restore backed up settings and data.</string>
-    <string name="settings_auto_restore_summary_usb">Note: Your %1$s needs to be plugged in for this to work.</string>
+    <string name="settings_auto_restore_summary">When recovering an app, also include settings and data.</string>
```

Comment:
reinstalling is the right term here, since all you need to do is re-install the app, you don't specfiically restore it, that's done automatically.

Reply:
My confusion: "Reinstalling" could mean the app would have to be re-downloaded, whereas "recovering", means the app is already there, and that in addition to this, the settings and data _could_ be included in the recovery of it.

In rare cases, I think confusion could arise over whether "settings and data" is fresh data, seeing as "recovery" is more closely tied to something existing, than "reinstalling", which could just overwrite it.

Reply:
It does have to be redownloaded, that is what this setting is for.

The way it works
1) You have a backup containing app AwesomeMusicPlayer
2) You have since uninstalled it
3) At a later date, you re-install AwesomeMusicPlayer (redownload needed)
4) If the backup is available, Seedvault will try to restore it's data automatically. hence, Automatic restore.

Reply:
I got confused with "re-install" meaning it not only has, but _is_ installed beforehand. I could be wrong.